src/components/RadioButton.js/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './style.module.css';
+
+
+function RadioButton({value, onChange,label,style, onBlur, name, children, className,checked, ...props}) {
+    const _className = classNames(styles.oc_radiobutton_container,styles.oc_field,className);
+    const _fieldClass = classNames(styles.oc_field_label,styles.oc_label, className)
+    const _wrapper = classNames(styles.oc_wrapper, className)
+    const _item = classNames(styles.oc_item, className)
+
+    return (
+        <div className={_wrapper}>
+            <div className={_item}>
+            <input 
+                className={_className}
+                type="radio"
+                name={name}
+                onChange={onChange}
+                onBlur={onBlur}
+                value={value}
+                checked={checked}
+                  {...props}
+            />
+            {children}
+        <label className={_fieldClass}style={{...style}} />
```

Comment:
This fails for [accessibility](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md), the label should be wrapped around the input

Reply:
The label cannot be wrapped inside the input it will throw an error

Reply:
![image](https://user-images.githubusercontent.com/20991583/70414349-25b1a780-1a5a-11ea-99f1-f69b651bcf5e.png)

![image](https://user-images.githubusercontent.com/20991583/70414392-3eba5880-1a5a-11ea-97b7-a0faf659ec14.png)

Reply:
<img width="272" alt="Screenshot 2019-12-09 at 8 30 47 AM" src="https://user-images.githubusercontent.com/39309699/70415988-3ebc5780-1a5e-11ea-9837-5c9bb665ae96.png">

this should be the right way

Reply:
Yeah